app/models/pet.rb

```diff
@@ -10,4 +12,13 @@ def shelter_name
   def self.adoptable
     where(adoptable: true)
   end
+
+  def self.filter_by_name(threshold)
+    where("name ILIKE ?", "%#{threshold}%")
+  end
+
+  def approved?
+    self.approved
+  end
```

Comment:
Nice job writing these method here. Remember that you will also need to test these methods in your `spec/models/pet_spec.rb"`